src/sentry/runner/commands/configoptions.py

```diff
@@ -203,20 +209,20 @@ def sync(ctx):
     from sentry.utils import metrics
 
     dry_run = bool(ctx.obj["dry_run"])
-    if dry_run:
-        click.echo("!!! Dry-run flag on. No update will be performed.")
 
     all_options = options.filter(options.FLAG_AUTOMATOR_MODIFIABLE)
 
     options_to_update = ctx.obj["options_to_update"]
     drift_found = bool(ctx.obj["drifted_options"])
+    presenter_delegator = ctx.obj["presenter_delegator"]
     for opt in all_options:
         if opt.name in options_to_update:
             try:
                 _attempt_update(
                     opt.name,
                     options_to_update[opt.name],
                     ctx.obj["drifted_options"],
+                    presenter_delegator,
```

Comment:
Please move the presenter at the beginning. 
There is a difference between the role of the existing parameters which are there for the business logic and the presenter which is there for presentation only so it would be a good idea to separate the two groups.